GOVERNANCE.md

```diff
@@ -0,0 +1,75 @@
+# OpenJS Standards Group
+
+<!-- yet to add -->
+For the current list of Team members, see the project [README.md](./README.md).
+
+## Members
+
+The [openjs-foundation/standards](https://github.com/openjs-foundation/standards) GitHub
+repository is maintained by the Team and additional Members who are
+added on an ongoing basis.
+
+* Invited to all meetings
+* Can participate in [consensus seeking process](#consensus-seeking-process)
+* Counted towards quorum in [Team Meetings](#team-meetings)
+* Participates in voting
+
+## Team Membership
+
+Team Membership is not time-limited. There is no fixed size of the Team.
+
+There is no specific set of requirements or qualifications for Team Membership beyond these rules.
+
+The following groups automatically qualify for membership and can request to be added to the GitHub team:
+
+* OpenJS Foundation CPC Members
+* OpenJS Foundation Project Maintainers
+
+## Team Meetings
+
+The Team meets bi-weekly on Zoom.us. A designated moderator
+approved by the Team runs the meeting. Each meeting should be
+published to YouTube.
```

Comment:
If the streaming doesn't work for any reasons, should the meeting be postponed?
It happened to one node.js meeting and to be 100% transparency we did it so